digitalocean/database/resource_database_cluster_test.go

```diff
@@ -882,6 +882,17 @@ resource "digitalocean_database_cluster" "foobar" {
   tags       = ["production"]
 }`
 
+const testAccCheckDigitalOceanDatabaseClusterOpensearch = `
+resource "digitalocean_database_cluster" "foobar" {
+  name       = "%s"
+  engine     = "opensearch"
+  version    = "%s"
+  size       = "db-s-2vcpu-2gb"
+  region     = "nyc1"
+  node_count = 3
+  tags       = ["production"]
```

Comment:
```suggestion
  size       = "db-s-1vcpu-2gb"
  region     = "nyc1"
  node_count = 1
  tags       = ["production"]
```

Not sure why, but looks like `db-s-2vcpu-2gb` isn't a supported size for Opensearch. While we're at it, we might as well use a single node configuration to keep the cost of running tests down.

```
$ doctl databases options slugs --engine opensearch
Slug              NodeNums
db-s-1vcpu-2gb    [1]
db-s-2vcpu-4gb    [1,3]
db-s-4vcpu-8gb    [1,3]
gd-2vcpu-8gb      [1,3,6,9,15]
gd-4vcpu-16gb     [1,3,6,9,15]
m3-2vcpu-16gb     [1,3,6,9,15]
m3-4vcpu-32gb     [1,3,6,9,15]
m3-8vcpu-64gb     [1,3,6,9,15]
```